lib/sidekiq/batch.rb

```diff
@@ -203,6 +203,7 @@ def process_successful_job(bid, jid)
       end
 
       def enqueue_callbacks(event, bid)
+        event = event.to_s
```

Comment:
Can you rename it to i.e. `event_name` or something. Shadowing method parameter is not the best practice.

Reply:
Sure, done.